core/src/main/java/io/questdb/griffin/SqlOptimiser.java

```diff
@@ -119,6 +119,8 @@ public class SqlOptimiser implements Mutable {
     private ObjList<JoinContext> emittedJoinClauses;
     private CharSequence tempColumnAlias;
     private QueryModel tempQueryModel;
+    private static final ObjList<String> AGGREGATE_FUNCTIONS_OPTIMISED_BY_BACKWARD_SCAN = new ObjList<>("LAST", "max");
```

Comment:
These lists look redundant: it's ok to have function names hardcoded in the checks instead of adding these fields.